router/src/server.rs

```diff
@@ -529,11 +529,19 @@ pub async fn run(
     // Create router
     let app = Router::new()
         .merge(SwaggerUi::new("/docs").url("/api-doc/openapi.json", ApiDoc::openapi()))
+        // Base routes
         .route("/", post(compat_generate))
         .route("/generate", post(generate))
         .route("/generate_stream", post(generate_stream))
-        .route("/", get(health))
+        // AWS Sagemaker route
+        .route("/invocations", post(compat_generate))
+        // Base Health route
```

Comment:
I think it would complexify things for no real gains. Maybe the runtime RAM would be a bit lower but I don't think it matters too much compared to the python servers used RAM.